src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -46,6 +47,15 @@ public SarifWorkItemFiler(Uri filingUri = null, SarifWorkItemContext filingConte
             this.FilingClient = FilingClientFactory.Create(this.FilingContext.HostUri);
 
             this.Logger = ServiceProviderFactory.ServiceProvider.GetService<ILogger<FilingClient>>();
+
+            Assembly currentAssembly = Assembly.GetExecutingAssembly();
```

Comment:
>currentAssembly [](start = 21, length = 15)

Throw this in a helper somewhere? Wouldn't this be useful to invoke for all the transformers as well? I think I suggest putting an extension method on the Assembly type. So you can do:

Assembly.GetExecutingAssembly().LogIdentity() or some such.